ledger/src/leader_schedule.rs

```diff
@@ -23,6 +27,15 @@ pub trait LeaderScheduleVariant:
 {
     fn get_slot_leaders(&self) -> &[Pubkey];
     fn get_leader_slots_map(&self) -> &HashMap<Pubkey, Arc<Vec<usize>>>;
+    fn is_vote_keyed(&self) -> bool {
```

Comment:
this seems to be test-only, can we limit its scope? typically taboo to impose a generic implementation to identify itself